pkg/infoschema/bundle_builder.go

```diff
@@ -112,8 +112,14 @@ func (b *bundleInfoBuilder) completeUpdateTables(is *infoSchema) {
 	}
 }
 
-func (b *bundleInfoBuilder) updateTableBundles(is *infoSchema, tableID int64) {
-	tbl, ok := is.TableByID(tableID)
+func (b *bundleInfoBuilder) updateTableBundles(infoSchemaInterface InfoSchema, tableID int64) {
+	// TODO: refactor
```

Comment:
You can implement this method

```
func (is *infoschema) base() *infoschema {
    return is
}
```

This both infoschema and infoschemav2 get it:

```
is := infoschemaInterface.base()
```